app/src/main/java/com/example/pocketcloset/Fragments/CameraFragment.java

```diff
@@ -106,9 +183,10 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
         //Detects request codes
         if(requestCode==GET_FROM_GALLERY && resultCode == Activity.RESULT_OK) {
             Uri selectedImage = data.getData();
-            Bitmap bitmap = null;
+           // Bitmap bitmap = null;
```

Comment:
nit: delete code instead of commenting out. You can use source history to restore if necessary.